libs/sdk-ui-kit/src/Dialog/HubspotConversionTouchPointDialogBase.tsx

```diff
@@ -34,6 +34,13 @@ export interface IHubspotFormField {
     click: () => void;
 }
 
+/**
+ * @internal
+ */
+export interface IHubspotJqueryFormField {
+    [key: string]: IHubspotFormField[] | string;
```

Comment:
According to the comment https://github.com/gooddata/gooddata-ui-sdk/pull/1824#discussion_r731454593, the type of `$form[0]` is not really an array but is an array-like object, I think we should redefine the type as below to reflect that fact, otherwise someone in the future might mistakenly call Array's functions like `push` or `shift`, causing the app to crash.

`[key: string]: IHubspotFormField[] | string;`

↓

`[key: string]: ArrayLike<IHubspotFormField> | string;`

Reply:
I have updated this. Thanks!